### PR TITLE
Disable callback url capture for windows as a temp workaround

### PR DESCRIPTION
**Pull request by janedegtiareva**
This needs a deeper dive to fix properly.

### CODE REVIEW

**Comment by janedegtiareva** [comment]
Tested on windows and through cloud VM terminal login flow (have to use the fallback).

**Comment by amgando** [comment]
works fine on macOS

File: index.js [hunk 1]
```diff
@@ -108,12 +117,13 @@ exports.login = async function (options) {
...
             output: process.stdout
         });
+        const redirectAutomaticallyHint = tempUrl ? '(if not redirected automatically)' : '';
         const getAccountFromConsole = async () => {
             return await new Promise((resolve) => {
                 rl.question(
                     chalk`Please authorize at least one account at the URL above.\n\n` +
                     chalk`Which account did you authorize for use with NEAR Shell?\n` +
-                    chalk`{bold Enter it here (if not redirected automatically):}\n`, async (accountId) => {
+                    chalk`{bold Enter it here${redirectAutomaticallyHint}:}\n`, async (accountId) => {
```
Comment (amgando) [inline index.js:126]: better to add whitespace after the `e` in `"here"`

Comment (amgando) [reply index.js:126]: otherwise it looks cramped
image

**Comment by amgando** [comment]
@AnaisUrlichs can you try this on linux?

```
git clone git@github.com:near/near-shell.git
cd near-shell
./bin/near login
```

does it:
- open a web browser to NEAR Wallet?
- move through the wallet flow until the page "you are logged in. close this window"?
- show something like this in your console when you switch back?

image

**Comment by AnaisUrlichs** [comment]
@janedegtiareva @amgando I receive the following error
Screenshot from 2020-05-01 09-12-34

**Comment by AnaisUrlichs** [comment]
The following works

git clone git@github.com:near/near-shell.git
cd near-shell
yarn
./bin/near login

**Comment by amgando** [comment]
@janedegtiareva CI tests are bc `config.js` doesn't have a `ci-staging` key.  see #308 for possible fix

**Pull request merged**